Sources/Tools/SoundDecompress/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+option(SoundDecompress_MOULa_Mode "Enable legacy MOULa mode in SoundDecompress" OFF)
```

Comment:
From the perspective of a user, I find myself wondering: what does "legacy MOULa mode" mean? Also, shouldn't this only be available on Windows?

Reply:
What kind of user are we talking about here?

Not sure how else to describe this; that's exactly its purpose, and anything more descriptive becomes too long for a single sentence description. :(

Reply:
I would word it as "Output wave files for MOULa"

Reply:
I suppose that addresses both "what it does" and "what it's for" reasonably well.